src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -92,9 +92,14 @@ Learn more about [routing alerts with integrations](/product/alerts/create-alert
 
 [Issue owners](/product/issues/issue-owners/) can receive notifications by email or Slack depending on the [notification settings](/product/alerts/notifications/notification-settings/) of the issue owner when an alert is triggered.
 
-If an issue owner is not configured or not found, either the notification won’t be sent or it’ll be sent to all project members, depending on the following setting in **[Project] > Settings > Issue Owners**.
+If an issue owner is not configured or not found, it will fallback to the teams or members specified in the fallback.
 
-![A toggle indicating if all users are issue owners or not.](issue_owners2.png)
+The following members will be notified based on the option selected and their notification settings:
+- All Project Members: All members that are on a team that has access to the project will be notified. This has the potential to notify a large number of people. If two teams have access to the project, all members of both teams will be notified.
+- Team Admins + Recently active members: Team admins of teams that have access to the project along with 10 of the most recently logged in members.
+- Nobody: If the issue has no issue owner, no one will be notified.
```

Comment:
yes! If they have picked "Nobody" in the fallback options, nobody gets notified

Reply:
Ah okay. Then I suggest taking out the first part of the sentence then because it sounds like we're indicating another condition, when the condition has already been met if you're using this setting:
```suggestion
- Nobody: No one will be notified.
```